frontend/src/app/modules/modal/modal.sass

```diff
@@ -27,6 +27,9 @@
     width: 70rem
     min-height: 40vh
 
+  &_fill_height
```

Comment:
maybe `autogrow` or `autoheight`? I'm okay with the current naming, but in that case it should be `fill-height`, so you don have a modifier on top of a modifier

Reply:
I changed it to `autoheight`